src/sentry/static/sentry/app/views/organizationEvents/events.jsx

```diff
@@ -121,5 +143,18 @@ const StyledSearchBar = styled(SearchBar)`
   flex: 1;
 `;
 
+const RowDisplay = styled('div')`
+  color: #afa3bb;
+  float: left;
+  margin: 20px 0 0 0;
```

Comment:
I don't think we'll need this or padding if we change parent component to flex - but if we do can we use `app/styles/space`?